utils.js

```diff
@@ -0,0 +1,60 @@
+import { canvas, ctx } from './canvas'
+
+let coord = {
+  x: 0,
+  y: 0
+}
+
+const getPosition = (event, coord) => {
+  coord.x = event.clientX - canvas.offsetLeft
+  coord.y = event.clientY - canvas.offsetTop
+}
+
+const drawShapes = shapes => {
+  shapes.forEach((arr, index) => {
+    switch (arr.type) {
```

Comment:
here you can also use functions from corresponding files: path.js, circle.js etc. to draw this shapes

Reply:
those functions have some differences, for example functions in path.js draw line and save data, but function in utils.js draw and read data